scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -254,6 +258,42 @@ function doImportEclipse() {
   "${ECLIPSE_HOME}/eclipse" -nosplash -vm "${ECLIPSE_JAVA_HOME}/bin/java" -application org.eclipse.ant.core.antRunner -buildfile "${DEVON_IDE_HOME}/scripts/lib_script/import.xml" -data "${WORKSPACE_PATH}" -keyring ~/.eclipse/.keyring -vm "${ECLIPSE_JAVA_HOME}/bin/java" -vmargs -DdevonImportPath="${importPath}" -DdevonImportWorkingSet="${importWorkingSets}" ${ECLIPSE_VMARGS} 
 }
 
+# $1: URL
+# $2: name
+# $3: optional dest
+function doMirrorUpdateSite() {
+  if [ -z "${2}" ]
+  then
+    doFail "Parameter <URL> or <name> is missing."
+  fi
+  local dest="${ECLIPSE_HOME}/update-sites/"
```

Comment:
Should we change the default destination of the mirrored update-sites? Maybe in conf?

Reply:
Yes, I would not put this into the Eclipse installation.
Maybe just use the download folder of devonfw-ide as default and create an updates-sites subfolder there.